app/views/catalog/_summary_data.html.erb

```diff
@@ -52,7 +52,7 @@
     <%= document[:summary_display].join('<br/>').html_safe %>
   </div>
 <%# TODO: Remove this elsif block after new mods index is in place %>
-<% elsif document.respond_to?(:mods) && document.mods.present? %>
+<% elsif document.respond_to?(:mods?) && document.mods? && document.mods.present? %>
```

Comment:
Elsewhere in this PR `document.mods?` replaces `document.mods.present?` but here they're both used. Intentional or cruft?

Reply:
👍 ; i think this check is unnecessary anyway... if there's mods, we'll be parsing it anyway 🤷